src/data/gen2Banner-urls.ts

```diff
@@ -29,57 +57,196 @@ export const GEN2BANNER_URLS = [
   '/javascript/tools/cli/graphqlapi/override-appsync/',
+  '/angular/tools/cli/graphqlapi/override-appsync/',
+  '/react/tools/cli/graphqlapi/override-appsync/',
+  '/nextjs/tools/cli/graphqlapi/override-appsync/',
+  '/vue/tools/cli/graphqlapi/override-appsync/',
   '/javascript/build-a-backend/graphqlapi/offline/',
+  '/react/build-a-backend/graphqlapi/offline/',
+  '/angular/build-a-backend/graphqlapi/offline/',
+  '/vue/build-a-backend/graphqlapi/offline/',
+  '/nextjs/build-a-backend/graphqlapi/offline/',
   '/javascript/build-a-backend/graphqlapi/best-practice/',
+  '/react/build-a-backend/graphqlapi/best-practice/',
+  '/angular/build-a-backend/graphqlapi/best-practice/',
+  '/vue/build-a-backend/graphqlapi/best-practice/',
+  '/nextjs/build-a-backend/graphqlapi/best-practice/',
   '/javascript/build-a-backend/graphqlapi/best-practice/deltasync/',
+  '/angular/build-a-backend/graphqlapi/best-practice/deltasync/',
+  '/react/build-a-backend/graphqlapi/best-practice/deltasync/',
+  '/nextjs/build-a-backend/graphqlapi/best-practice/deltasync/',
+  '/vue/build-a-backend/graphqlapi/best-practice/deltasync/',
   '/javascript/build-a-backend/graphqlapi/best-practice/create-blog/',
-  '/javascript/build-a-backend/graphqlapi/best-practice/wharehouse-management/',
+  '/angular/build-a-backend/graphqlapi/best-practice/create-blog/',
+  '/react/build-a-backend/graphqlapi/best-practice/create-blog/',
+  '/nextjs/build-a-backend/graphqlapi/best-practice/create-blog/',
+  '/vue/build-a-backend/graphqlapi/best-practice/create-blog/',
+  '/javascript/build-a-backend/graphqlapi/best-practice/warehouse-management/',
+  '/angular/build-a-backend/graphqlapi/best-practice/warehouse-management/',
+  '/react/build-a-backend/graphqlapi/best-practice/warehouse-management/',
+  '/nextjs/build-a-backend/graphqlapi/best-practice/warehouse-management/',
+  '/vue/build-a-backend/graphqlapi/best-practice/warehouse-management/',
   '/javascript/tools/cli/graphqlapi/search-and-result-aggregations/',
+  '/react/tools/cli/graphqlapi/search-and-result-aggregations/',
+  '/angular/tools/cli/graphqlapi/search-and-result-aggregations/',
+  '/nextjs/tools/cli/graphqlapi/search-and-result-aggregations/',
+  '/vue/tools/cli/graphqlapi/search-and-result-aggregations/',
   '/javascript/tools/cli/graphqlapi/connect-machine-learning-services/',
+  '/angular/tools/cli/graphqlapi/connect-machine-learning-services/',
+  '/react/tools/cli/graphqlapi/connect-machine-learning-services/',
+  '/nextjs/tools/cli/graphqlapi/connect-machine-learning-services/',
+  '/vue/tools/cli/graphqlapi/connect-machine-learning-services/',
   '/javascript/build-a-backend/graphqlapi/troubleshooting/',
+  '/react/build-a-backend/graphqlapi/troubleshooting/',
+  '/angular/build-a-backend/graphqlapi/troubleshooting/',
+  '/vue/build-a-backend/graphqlapi/troubleshooting/',
+  '/nextjs/build-a-backend/graphqlapi/troubleshooting/',
   '/javascript/tools/cli/graphqlapi/directives-reference/',
```

Comment:
Just note this one is an exception compared to the others and is good as is under CLI